viewmaster/pdsgrouptable.py

```
@@ -1,15 +1,40 @@
+    PdsGroupTable organizes groups of related PdsFiles into a single table
+    section. All groups must share the same parent directory. Groups can be
+    hidden, in which case the default iterator excludes them.
+
+    Attributes:
+        parent_pdsf: Parent `Pds3File` (False until initialized; None for
+            merged directories).
+        groups (list): Ordered list of `PdsGroup` objects.
+        _levels_filled (list|None): Cached parent hierarchy levels.
+    """
 
     def __init__(self, pdsgroups=[], parent=False):
```

Ruff (0.14.5)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)